docs/app/ComponentGuidelines.md

```diff
@@ -190,28 +190,28 @@ All magic is noted in the documentation examples.
 
 ```jsx
 <Form.Field className='inherit-this' />
-// => <div className='sd-field inherit-this field>...
+// => <div className='sd-form-field inherit-this field>...
```

Comment:
Components were reorganized to sub components in #99:
`Field` => `FormField`.

Generated `className`s should also follow suit:
`className='sd-field'` => `className='sd-form-field'`

Fixing conformance tests pointed this out.